docs/framework/winforms/disable-dpi-awareness-visual-studio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Disable DPI-awareness in Visual Studio
+ms.date: 08/14/2018
+ms.prod: visual-studio-dev15
+ms.technology: vs-ide-designers
+ms.topic: conceptual
+author: gewarren
+ms.author: gewarren
+manager: douge
+---
+# Disable DPI-awareness in Visual Studio
+
+Visual Studio is a dots per inch (DPI) aware application, which means displays scale automatically. If an application states that it's not DPI-aware, the operating system scales the application as a bitmap. This behavior is also called DPI virtualization. The application still thinks that it's running at 100% scaling, or 96 dpi.
+
+## Rendering issue
+
+The **Windows Forms Designer** in Visual Studio doesn't currently have scaling support. This can cause display issues when you open forms in the **Windows Forms Designer** on HDPI monitors.
+
+For example, the first image that follows shows **Windows Forms Designer** on a monitor at 100% scaling. The second image shows what the form looks like when it's opened on a high dots per inch (HDPI) monitor:
+
+![Windows Forms Designer scaling at 100%](media/scaling-100-percent.png)
+
+![Windows Forms Designer on HDPI monitor](media/win-forms-designer-hdpi.png)
+
+In Visual Studio 2017 version 15.8 and later, when you open a form in the **Windows Forms Designer**ga . on an HDPI monitor, Visual Studio displays a yellow message bar at the top of the designer:
+
+![Scaling on your monitor yellow message bar in Visual Studio](media/scaling-gold-bar.png)
+
+The message reads **Scaling on your main display is set to 200%. This might cause rendering problems in the designer window.**
+
+## To resolve the problem
+
+There are two options to resolve the display problem:
+
+- Set your display scaling setting to 100%.
```

Comment:
Restarting the VS is more preferred scenario because setting scaling to 100% usually results to very small unusable UI. That's why I would put this section first. Something like this:

There are two options to resolve the display problem:
1. Restart Visual Studio as a DPI-unaware process. This will fix the designer issues but may result in a slightly blurry fonts. To do so click on the option **Restart Visual Studio as a DPI-unaware process** on the yellow informational bar. After Visual Studio is restarted, you will see a different information bar that says "...".
<img width="1010" alt="goldbar_secondscreen1" src="https://user-images.githubusercontent.com/31997351/43943189-794a6d50-9c2f-11e8-9c55-c05f4ddcf077.png">
This allows you to go back DPI-aware mode when you are done working with the designer, so the fonts don't look blurry and you don't encounter issues with WPF and XAML designers. To go back to DPI-aware mode, clicking on "Restart Visual Studio as a DPI-aware process".

2. Set your display scaling setting to 100%.

To do this ...